tests/integration/helpers.bash

```diff
@@ -448,6 +448,13 @@ function requires() {
 				skip_me=1
 			fi
 			;;
+		psi)
+			# If PSI is not compiled in the kernel, the file will not exist.
+			# If PIS is compiled, but not enable, read will fail with ENOSUPP
+			if [[ ! $(cat /sys/fs/cgroup/cpu.pressure) ]]; then
```

Comment:
You don't need to `cat` the file, mere existence check should be sufficient.

IOW
```bash
if [ ! -f /sys/fs/cgroup/cpu.pressure ]; then
...
```

Reply:
Ah, I see, you *do* need to actually cat the file. I should have read better before writing comments 🤦🏻